### DISCUSSION TITLE
Saving Recent changes to TodayNote

### DISCUSSION THREAD

**Discussion by Vantomas** [TriliumNext/Trilium#3716] [Q&A] [answered]
Hello,
I want to every day at midnight take a list of all the changed notes on that day and save them in TodayNote for history reference. A list like in the Recent changes popup is absolutely sufficient. 

Can you point me in the direction of how I can archieve this? Is there any API access to Recent changes? Or I have to use the api.searchForNotes and generate changelog manually?

**Comment by zadam** [comment, marked as answer]
Hi, have you noticed "Edited Notes" on day notes? It does basically what you want and is compiled from the same data as recent changes (note created/modified dates + revisions). 

<img width="125" alt="image" src="https://user-images.githubusercontent.com/617641/225362144-015fef45-42ea-4632-9e15-14f4af67dd47.png">


If you want to store it really in notes, then there's a private REST API for both edited notes and recent changes.

**Reply by Vantomas** [reply]
Oh gosh, I feel so silly, I really didn't notice that. This will do just fine, thank you so much.
